### DISCUSSION TITLE
✔️ Add version check on start

### DISCUSSION THREAD

**Discussion by kevingrillet** [zebscripts/AFK-Daily#43] [Ideas]
If there is a new release version, by using this [gist](https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c) it could be possible to display a warning for a new version.

Or just add a file `version.ini` that could be curl directly and compare to the local file:
```shell
function get_latest_version() {
  curl --silent "https://raw.githubusercontent.com/zebscripts/AFK-Daily/master/version.ini" # Get latest version file
}
function version_check() {
  if [ `cat version.ini` != `get_latest_version` ] ; then
    echo "[WARN] New version available, please download the last release."
  fi
}
```

**Comment by Zebiano** [comment]
That is a very interesting gist, thank you for the share! Though I wonder why I'd want to use it? The script already checks if there is an update available, and automatically updates if possible.

**Reply by kevingrillet** [reply]
It's a secondary way to check if the user did not clone the repo, but downloaded the zip file, or use it in virtual environments without git installed.

**Reply by Zebiano** [reply]
Right, that does indeed make sense!

**Reply by Zebiano** [reply]
Though for now, this will be a low priority as a check is implemented for those who do `clone` the repo. And since that is the intended way for anyone to install the script, I'm going to pretend they actually do so. If not, automatic updates is not an available feature for now.
